packages/big-design/src/components/StatelessPagination/StatelessPagination.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  ArrowDropDownIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from '@bigcommerce/big-design-icons';
+import React, { memo } from 'react';
+
+import { MarginProps } from '../../helpers';
+import { Dropdown, DropdownItem } from '../Dropdown';
+import { Flex, FlexItem } from '../Flex';
+
+import { StyledButton } from './styled';
+
+export interface StatelessPaginationLocalization {
+  previousPage: string;
+  nextPage: string;
+}
+
+const defaultLocalization: StatelessPaginationLocalization = {
+  previousPage: 'Previous page',
+  nextPage: 'Next page',
+};
+
+export interface StatelessPaginationProps extends MarginProps {
+  itemsPerPage: number;
+  itemsPerPageOptions: number[];
+  onItemsPerPageChange(range: number): void;
+  disableNext?: boolean;
+  onNext(): void;
+  disablePrevious?: boolean;
+  onPrevious(): void;
+  label?: string;
+  rangeLabel?: string;
+  localization?: StatelessPaginationLocalization;
```

Comment:
🤔 What makes `label` and `rangeLabel` special in comparison to `previousPage`/`nextPage`?

Reply:
other than `rangeLabel` being a touch more volatile (given the limit can changed), not a lot

I've moved them all into a consolidated `locaization` object: https://github.com/bigcommerce/big-design/pull/1598/commits/7e2fe591649c9c0eea33fa87816fd92b5059c7f5

it add a _tiny_ bit more complexity to the implementation, now that it's fully optional and one default is dynamic